### PR TITLE
Add types on module files

### PR DESCRIPTION
**Pull request by ooflorent**
**What kind of change does this PR introduce?**

types

**Did you add tests for your changes?**

no

**Does this PR introduce a breaking change?**

no

**What needs to be documented once your changes are merged?**

nothing

### CODE REVIEW

File: lib/ModuleRestoreError.js [hunk 1]
```diff
@@ -21,7 +27,7 @@ class ModuleRestoreError extends WebpackError {
 				message += err;
 			}
 		} else {
-			message = err;
+			message = String(err);
```
Comment (sokra) [inline lib/ModuleRestoreError.js]:
```suggestion
			message += String(err);
```

File: lib/ModuleBuildError.js [hunk 2]
```diff
@@ -38,10 +42,10 @@ class ModuleBuildError extends WebpackError {
 			} else if (typeof err.message === "string" && err.message) {
 				message += err.message;
 			} else {
-				message += err;
+				message += String(err);
```
Comment (ooflorent) [inline lib/ModuleBuildError.js]: Shouldn’t the message be prepended by `:`?

Comment (sokra) [reply lib/ModuleBuildError.js]: This is done in line 21 - 25

Comment (ooflorent) [reply lib/ModuleBuildError.js]: Right. Looks like it's ready to ship!

**Comment by sokra** [comment]
Thanks

**Pull request merged**